quickwit/quickwit-storage/tests/google_cloud_storage.rs

```diff
@@ -25,33 +25,23 @@ mod gcp_storage_test_suite {
     use quickwit_common::setup_logging_for_tests;
     use quickwit_common::uri::Uri;
     use quickwit_storage::test_config_helpers::{
-        LOCAL_GCP_EMULATOR_ENDPOINT, new_emulated_google_cloud_storage,
+        DummyTokenLoader, LOCAL_GCP_EMULATOR_ENDPOINT, new_emulated_google_cloud_storage,
     };
 
     pub async fn sign_gcs_request(req: &mut reqwest::Request) -> anyhow::Result<()> {
```

Comment:
Disregard all my previous comments. It seems like we can rewrite `sign_gcs_request` as the following and tests should pass:
```rust
pub fn sign_gcs_request(req: &mut reqwest::Request) -> anyhow::Result<()> {
        req.headers_mut().insert(
            reqwest::header::AUTHORIZATION,
            reqwest::header::HeaderValue::from_str("Bearer dummy")?,
        );
        Ok(())
    }
```